### PR TITLE
[bengio-w12] 6L/256d depth extension + T_max=50 (3rd attempt)

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

Extending depth from 5L→6L on the current best architecture (5L/256d + FourierEmbed + T_max=50) will yield further improvement beyond the already-validated 4L→5L gain.

**Rationale**: The 4L→5L upgrade (alphonse PR #174) delivered −0.254pp (7.2091% → 6.9549%). This is consistent with depth scaling in transformer-based neural operators — each additional layer increases the model's capacity to capture complex nonlinear feature interactions in the CFD field. The wall-shear axes (wsy=8.73%, wsz=10.58%) are the primary binding constraint and likely require deeper feature hierarchies to resolve. Adding a 6th layer increases n_params to ~3.9M (+0.65M from cross-attention FFN blocks at 256d).

All other hyperparameters remain fixed at the validated best: FourierEmbed PE, T_max=50 cosine schedule, lr=3e-4, no-EMA.

**This is a third-attempt assignment.** Previous assignments for this hypothesis (PR #413, PR #426) were closed for non-compliance (zero ACK). This is the final attempt. Failure to acknowledge within 30 minutes will result in permanent reassignment of this hypothesis.

## Instructions

Run a single training experiment:

```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 6 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 50 \
  --no-compile-model \
  --wandb-group bengio-wave12-depth
```

**Gate schedule** (report results in PR comments at each gate):
- ep5: `val_primary/abupt_axis_mean_rel_l2_pct` ≤ 11.0% (kill if not below)
- ep15: ≤ 8.9% (kill if not below)
- ep30: ≤ 7.5% (kill if not below)
- ep50: final result — target ≤ 6.90% (beat baseline 6.9549%)

**Critical flags** (ALL required):
- `--fourier-pe` — mandatory for comparability (n_params must be ~3.9M for 6L)
- `--no-compile-model` — mandatory (PyTorch inductor crash at validation)
- `--model-heads 4` — maintain 4H consistent with baseline

**ACK required within 30 minutes** by posting a comment on this PR with your W&B run ID.

## Baseline

Current best (PR #174, alphonse, bengio branch):

| Metric | Current Best (val) | AB-UPT Target |
|--------|--------------------|---------------|
| `val_primary/abupt_axis_mean_rel_l2_pct` | **6.9549%** | 4.51% |
| `val_primary/surface_pressure_rel_l2_pct` | 4.5644% | 3.82% |
| `val_primary/volume_pressure_rel_l2_pct` | 3.9361% ✓ | 6.08% |
| `val_primary/wall_shear_y_rel_l2_pct` | 8.7345% | 3.65% |
| `val_primary/wall_shear_z_rel_l2_pct` | 10.5766% | 3.63% |

Best checkpoint: ep~45.3, step 807,025, W&B run `vu4jsiic`
Architecture: 5L/256d/4H + FourierEmbed + T_max=50, n_params=3,249,813

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #432 because **Closed — third-strike zero-ACK violation.**

PR #432 was assigned 2026-05-02T17:16Z. The 30-minute ACK deadline expired at 17:46Z with zero comments from the student.

This is the third consecutive zero-ACK failure for this hypothesis (PR #413, PR #426, PR #432). This hypothesis (6L/256d depth extension + T_max=50) will be permanently reassigned to a different student.

alphonse's GPU is now released for reassignment..
